src/apps/relay/dtls_listener.c

```diff
@@ -130,6 +133,33 @@ int get_dtls_version(const unsigned char *buf, int len) {
   return 0;
 }
 
+static size_t print_packet_txt2pcap(uint64_t now, uint8_t *payload, size_t payload_length, uint8_t *txt2pcap,
+                                    size_t txt2pcap_length) {
+
+  size_t index = 0;
+  int64_t remaining = now % (24 * 60 * 60 * 1000);
```

Comment:
unsigned to signed conversion. 

Is there any benefit to converting to signed, instead of staying unsigned?